src/bundle/Resources/public/js/alloyeditor/src/plugins/ez-embed.js

```diff
@@ -21,9 +21,7 @@ const ZERO_WIDTH_SPACE = '&#8203;';
         init: function(editor) {
             editor.ezembed = {
                 canBeAdded: () => {
-                    const path = editor.elementPath();
-
-                    return !path || path.contains('table', true) === null;
+                    return true;
```

Comment:
This method is not needed anymore so we can remove it in 3.0 for BC we need to keep it in 2.5.x.
Please add the deprecation information:
```js
console.warn('[DEPRECATED] canBeAdded method is deprecated');
console.warn('[DEPRECATED] it will be removed from ezplatform-admin-ui 2.0');
```

The usage of this method can be safely removed from our buttons:
https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-image.js#L60 and https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-embed.js#L58
Please remove the disable state and the method `isDisabled`

Reply:
All requested changes are done. Also a deprecation warning for `EzBtnImage.isDisabled` and `EzBtnEmbed.isDisabled` was added: https://github.com/ezsystems/ezplatform-admin-ui/pull/1034/commits/8b3687bcf93f05e3d5c7cd3fcffd1d0a831eb48b